AzureAppServiceSetAppSettings/Tasks/ConnectionStrings/ApplyConnectionStrings.ps1

```diff
@@ -22,12 +22,13 @@ param(
 import-module "Microsoft.TeamFoundation.DistributedTask.Task.Internal"
 import-module "Microsoft.TeamFoundation.DistributedTask.Task.Common"
 
-$useSlot = $Slot -ne ""
-$slotLabel = If ($useSlot) { $Slot } Else { "<none>" }
+If ($Slot -eq "") {
```

Comment:
This becomes unnecessary if you use "production" as the default value for the $Slot paramater up to.